packages/opentelemetry-plugin-https/package.json

```diff
@@ -42,7 +42,6 @@
   },
   "devDependencies": {
     "@opentelemetry/node": "^0.4.0",
-    "@opentelemetry/scope-base": "^0.4.0",
```

Comment:
Can we remove this from `@opentelemetry/http` package?

Reply:
It is a devDependency in the `http` package which is only used for tests (noop scope manager).